src/Tests/OneExplorer/ArtifactLocator.test.ts

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {assert} from 'chai';
+import * as fs from 'fs';
+import * as vscode from 'vscode';
+import {Artifact, ArtifactAttr, Locator, LocatorRunner} from '../../OneExplorer/ArtifactLocator';
+
+// TODO Extract as test fixture
+class TestBuilder {
```

Comment:
I think this class is not used for this test file... is there something I missed?

Reply:
I will remove it. ;-D